src/components/SearchArea.module.css

```
@@ -76,7 +76,7 @@
   margin-left: 1px;
   margin-top: 0px;
   font-size: 15px;
-  font-family: "Source Sans Pro";
+  font-family: "Inter", sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)